libs/providers/flagd/src/lib/flagd-provider.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  EvaluationContext,
+  Provider,
+  ResolutionDetails,
+  ProviderOptions
+} from '@openfeature/nodejs-sdk';
+import {IService} from './service/IService'
+import HTTPService from './service/http/service'
+
+export interface FlagdProviderOptions extends ProviderOptions {
```

Comment:
Why extend provider options? It doesn't look only the service is being used.

Reply:
have removed the extend, is now standalone interface